components/02-elements/tab/tab.hbs

```diff
@@ -16,7 +14,12 @@
              "
              data-content="{{ tabId }}"
         >
-            {{{ content }}}
+            {{#if content}}
+                {{{ content }}}
+            {{/if}}
+            {{#if contentElement}}
```

Comment:
just use `{{ else }}`

Reply:
I use only one in one time.

Reply:
ok, as we discussed on chat, keep 2 if statements